### PR TITLE
Fixes #205

### LINKED ISSUE #205: IndexError while setting input in BinaryAdder
```
>>> from __future__ import print_function
>>> from BinPy.Combinational.combinational import *
>>> ba = BinaryAdder([0, 1], [1, 0], 0) # Input is of the form ([Binary number 1],[ BInary number 2], Carry]
>>> print(ba.output()) # Output will be of the form [ [SUM], CARRY]
[0, 1, 1]
>>> # Changing input: ba.setInput(1, [0]) #Input at index 1 is changed to 0
>>> ba.setInput(1, [0])
>>> print (ba.output())
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
<ipython-input-8-7b2cdcae6f07> in <module>()
      1 # Changing input: ba.setInput(1, [0]) #Input at index 1 is changed to 0
----> 2 ba.setInput(1, [0])
      3 print (ba.output())

/usr/local/lib/python2.7/dist-packages/BinPy-0.3-py2.7.egg/BinPy/Gates/gates.pyc in setInput(self, index, value)
     87         if isinstance(value, Connector):
     88             value.tap(self, 'input')
---> 89         self.trigger()
     90 
     91     def getInputStates(self):

/usr/local/lib/python2.7/dist-packages/BinPy-0.3-py2.7.egg/BinPy/Combinational/combinational.pyc in trigger(self)
    124         for i in range(self.size - 1, -1, -1):
    125             S, carry = FullAdder(
--> 126                 self.inputs[0][i], self.inputs[1][i], carry).output()
    127             result.append(S)
    128         result.append(carry)

IndexError: list index out of range
```

### PR DESCRIPTION
**Pull request by mishravikas**

### CODE REVIEW

File: BinPy/Combinational/combinational.py [hunk 1]
```diff
@@ -339,8 +345,7 @@ def trigger(self):
         result = []
         borrow = self.borrow
         for i in range(self.size - 1, -1, -1):
-            D, borrow = FullSubtractor(
-                self.inputs[0][i], self.inputs[1][i], borrow).output()
+            D, borrow = FullSubtractor(self.inputs[0][i], self.inputs[1][i], borrow).output()
```
Comment (debugger22) [inline BinPy/Combinational/combinational.py]: Why did you do this?

File: BinPy/Combinational/combinational.py [hunk 2]
```diff
@@ -122,8 +122,14 @@ def trigger(self):
         result = []
         carry = self.carry
         for i in range(self.size - 1, -1, -1):
-            S, carry = FullAdder(
-                self.inputs[0][i], self.inputs[1][i], carry).output()
+            # whenever setInput() function is called and input's size is less than the self.size it raises IndexError hence try and exeption here
```
Comment (debugger22) [inline BinPy/Combinational/combinational.py]: Put this comment in two lines

```
# Whenever setInput() function is called and input's size is less than
# the self.size it raises IndexError hence try and exeption here
```

**Comment by debugger22** [comment]
:+1: If these two things are done.

File: BinPy/Combinational/combinational.py (hunk 1 above)
Comment (debugger22) [inline BinPy/Combinational/combinational.py]: Ok it's not a big deal.

Comment (coder006) [inline BinPy/Combinational/combinational.py]: @mishravikas @debugger22 according to PEP 8 length of line should not exceed 72 characters. This one contains 81 even if you neglect white-spaces. Earlier code was fine.

File: BinPy/Combinational/combinational.py (hunk 2 above)
Comment (coder006) [inline BinPy/Combinational/combinational.py]: As i said...follow PEP 8 convention.

**Comment by debugger22** [comment]
Thanks! This is in.

**Pull request merged**